### ISSUE TITLE
index out of bounds exception thrown while logging to scans

### ISSUE THREAD

**Issue by ZacSweers** [cdsap/InfoKotlinProcess#20]
Version 0.1.4

```
WARNING: Error invoking build scan buildFinished action
--
  | java.lang.IndexOutOfBoundsException: Index -1 out of bounds for length 18
  | at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:100)
  | at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:106)
  | at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:302)
  | at java.base/java.util.Objects.checkIndex(Objects.java:385)
  | at java.base/java.util.ArrayList.get(ArrayList.java:427)
  | at io.github.cdsap.kotlinprocess.parser.JStatData.getCollector(JStatData.kt:59)
  | at io.github.cdsap.kotlinprocess.parser.JStatData.process(JStatData.kt:25)
  | at io.github.cdsap.kotlinprocess.ConsolidateProcesses.consolidate(ConsolidateProcesses.kt:14)
  | at io.github.cdsap.kotlinprocess.InfoKotlinProcessPlugin$buildScanReporting$1.execute(InfoKotlinProcessPlugin.kt:38)
  | at io.github.cdsap.kotlinprocess.InfoKotlinProcessPlugin$buildScanReporting$1.execute(InfoKotlinProcessPlugin.kt:37)
  | at com.gradle.enterprise.gradleplugin.internal.extension.a$5.a(SourceFile:213)
  | at com.gradle.enterprise.gradleplugin.internal.extension.a$5.accept(SourceFile:210)
  | at com.gradle.scan.plugin.internal.q.b.b.a(SourceFile:45)
  | at java.base/java.lang.Iterable.forEach(Iterable.java:75)
```

**Comment by cdsap** [comment]
Thanks for reporting this issue. 
 I tried to reproduce it, but always the jstat outputs returned the `CGC` property(the one that is causing the exception). My tests were in mac/linux with jdks 11, 17 and 19 for different vendors. 
Finally, I reworked how the output is parsed and now we are getting the relevant properties for the output. Released a new version 0.1.5. 
Can you confirm that is fixed please?

**Comment by ZacSweers** [comment]
Thanks! Will report back if we see it again after the update 👍
